client/app/main/student/student.controller.js

```diff
@@ -14,6 +14,16 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr, Modal) {
     $scope.student = result.student;
     $scope.interventions = result.interventions;
     $scope.outreaches = result.outreaches;
+    Student.list({id: $stateParams.id}, function(data) {
+      var record = data.currentRecord.entries;
+      var enrolled = 0;
+      var present = 0;
+      _.forEach(record, function(entry) {
```

Comment:
We probably don't want to round down to the whole number. 

For example, if a student is present for 181 days of 200. They will have 90.5 percent attendance. If we use Math.floor(enrolled/present*100) their percent attendance would be displayed at 90 percent, which should categorize them as at risk. We should probably keep at least one extra sig fig.